src/package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "@mojaloop/sdk-standard-components",
-  "version": "10.6.0",
+  "version": "10.6.1",
```

Comment:
bump it to 10.6.2 and let first merge the fix: https://github.com/mojaloop/sdk-standard-components/pull/72